AnnService/inc/SSDServing/VectorSearch/SearchSsdIndex.h

```diff
@@ -531,13 +537,15 @@ namespace SPTAG {
 
                 if (p_opts.m_rerank > 0 && vectorSet != nullptr) {
                     LOG(Helper::LogLevel::LL_Info, "\n Begin rerank...\n");
+                    COMMON::DistanceUtils::Quantizer.reset();
+                    auto distCalc = SPTAG::COMMON::DistanceCalcSelector<ValueType>(COMMON_OPTS.m_distCalcMethod);
```

Comment:
This part has not changed? It can be replaced by the COMMON::DistanceUtils::ComputeDistance().